PyInstaller/building/makespec.py

```diff
@@ -100,6 +100,11 @@ def __add_options(parser):
                         "(default: first script's basename)")
 
     g = parser.add_argument_group('What to bundle, where to search')
+    g.add_argument('--datas',
```

Comment:
Should be singular, since this adds *one* entry to the list. `datas` is wrong English anyway - using it within the code is just a heritage.

(Using the (wrong) `datas` as destination is okay, though, as this is how it is used within the code.)